### DISCUSSION TITLE
scroll in ComboBox

### DISCUSSION THREAD

**Discussion by GeTechG** [emilk/egui#834] [Q&A]
I put a ComboBox in the ScrollArea. It's ok only the dropdown menu is cut off by this scroll, help me fix it.
![изображение](https://user-images.githubusercontent.com/20752579/138561796-102ff2cb-afb1-40c1-8520-8fc6d6b89097.png)

**Comment by emilk** [comment]
This is a known issue: https://github.com/emilk/egui/issues/825 (see particularly https://github.com/emilk/egui/issues/825#issuecomment-947747418)
